<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сравнение нетлистов</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>netcmp.ico</normaloff>netcmp.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="leftNetlist">
        <property name="placeholderText">
         <string>Первый нетлист</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="leftBrowse">
        <property name="text">
         <string>Обзор...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rightNetlist">
        <property name="placeholderText">
         <string>Второй нетлист</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rightBrowse">
        <property name="text">
         <string>Обзор...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="showDifferenceOnly">
        <property name="text">
         <string>Показывать только различия</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="font">
        <property name="text">
         <string>Шрифт...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="print">
        <property name="text">
         <string>Печать...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeView" name="netCmpView">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
